database/cloudinary/getImagesList.js

```diff
@@ -7,11 +7,21 @@ cloudinary.config({
   api_secret: cloudinaryKeys.api_secret
 });
 
-cloudinary.api.resources((err, res) => {
-  if (err) {
-    console.error(err);
-  } else {
-    let list = res.resources.map(info => info.secure_url);
-    console.log(list);
-  }
-})
+// How many results per call (Max 500)
+const imgUrlsPerCall = 10;
```

Comment:
Could you make this an input for your function for modularity and ease of use?

Reply:
Good call. I thought I had done that lol. I'll have to go back and fix it. Thanks!